### PR TITLE
[PHEE-594] fix mojaloop testcases

### PR DESCRIPTION
**Pull request by Anover000**
[PHEE-594 fix mojaloop testcases](https://fynarfin.atlassian.net/browse/PHEE-594?atlOrigin=eyJpIjoiMjlkMjBiNGY3NmNlNDk2NGJjY2Q0YWU0ZDBjMDJmOWUiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!
- [x] Followed the PR title naming convention mentioned above.

- [ ] Acknowledge that we will not merge PRs that are not passing the checks ("green") - it is your (author's) responsibility to get a proposed PR to pass all the checks, not primarily the project's maintainers.

- [ ] The PR title should include a JIRA ticket

- [ ] Design-related bullet points or design document links related to this PR added in the description above.

- [ ] Updated corresponding Postman Collection or API documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

- [ ] Followed coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

FYI our guidelines for code reviews same as https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -315,15 +315,15 @@ workflows:
...
-            - deploying-bpmns
+            - test-chart-gov
           context:
             - AWS
             - Helm
             - slack
             - Ngrok
       - test-chart-gov:
           requires:
-            - test-chart-ams
```
Comment (fynmanoj) [inline .circleci/config.yml:326]: Please don't change this order

File: src/test/java/resources/mojaloop.feature [hunk 2]
```diff
@@ -118,14 +118,15 @@ Feature: Mojaloop test
...
+    Given I have Fineract-Platform-TenantId for "payer"
     And I generate clientCorrelationId
     And I have private key
     And I generate signature
     When I call the batch transactions endpoint with expected status of 202
     And I am able to parse batch transactions response
     And I fetch batch ID from batch transaction API's response
     Then I will sleep for 10000 millisecond
-    When I call the batch aggregate API with expected status of 200 with total 2 txns
+    When I call the batch aggregate API with expected status of 200 with total 3 txns
```
Comment (fynmanoj) [inline src/test/java/resources/mojaloop.feature:129]: Are we adding additional transaction?

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (Anover000) [reply .circleci/config.yml:326]: done it temporarily to test, will reverse and notify.

File: src/test/java/resources/mojaloop.feature (hunk 2 above)
Comment (Anover000) [reply src/test/java/resources/mojaloop.feature:129]: No, earlier it was 3 only, not sure why it was changed to 2.

File: .circleci/config.yml (hunk 1 above)
Comment (Anover000) [reply .circleci/config.yml]: reversed

**Review by fynmanoj — APPROVED** [review]

**Pull request merged**